services/web-app/components/demo-links/demo-links.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Link as CarbonLink } from '@carbon/react'
+import PropTypes from 'prop-types'
+
+const DemoLinks = ({ links }) => {
+  let linkList = '–'
+  if (links) {
+    linkList = (
+      <>
```

Comment:
I think not having a key here is causing this issue 
<img width="842" alt="image" src="https://user-images.githubusercontent.com/40550942/158625254-0faa5bfe-7007-4b67-825c-5a2d8a9cc3a3.png">

Reply:
I'm seeing this error on main branch also, not sure if its related to these changes? I have a key on line 17 🤔

Reply:
got it, updated, also updated the component name since it won't be used for only demo links. (still wip adding Matt's other todos)